account_invoice_import/tests/test_invoice_import.py

```diff
@@ -1,25 +1,28 @@
 # Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+from odoo import fields
 from odoo.tests.common import TransactionCase
-from odoo.tools import float_compare
 
 
 class TestInvoiceImport(TransactionCase):
     def setUp(self):
         super().setUp()
```

Comment:
any chance to stop using `setUp` and use `setUpClass`?

Reply:
done by another merge PR